client/src/App.css

```diff
@@ -573,6 +576,10 @@ fieldset input {
     margin-left: 7%;
   }
 
+  .filter-content-label{
+    font-size:0.8rem
+  }
+
   @media(max-width:800px){
```

Comment:
Suggestion: Try to make separate CSS files for separate components. This way it is easier to manipulate them later